docs/examples/promise/02_listen_to_blocks/index.js

```diff
@@ -9,14 +9,15 @@ async function main () {
 
   // Subscribe to the new headers on-chain. The callback is fired when new headers
   // are found, the call itself returns a promise with a subscription that can be
-  // used to unsubscribe from the newHead subscription
+  // used to unsubscribe from the newHead subscription.
+  // Use the RPC Node Interface
   const subscriptionId = await api.rpc.chain.newHead((header) => {
     console.log(`best #${header.blockNumber}`);
   });
 
   // Id for the subscription, we can cleanup and unsubscribe via
   // `api.chain.newHead.unsubscribe(subscriptionId)`
-  console.log(`subsciptionId: ${subscriptionId}`);
+  console.log(`subscriptionId: ${subscriptionId}`);
 }
 
-main().catch(console.error);
+main().catch(console.error).finally(_ => process.exit());
```

Comment:
Like the README said, listens until Ctrl-C. (If we do this it will just grab the first header, not show them as they come in. So revert to original, it is correct.)